contracts/access/TimelockController.sol

```diff
@@ -269,9 +271,14 @@ contract TimelockController is AccessControl {
     }
 
     /**
-     * @dev Changes the timelock duration for future operations.
+     * @dev Changes the minimum timelock duration for future operations.
      *
      * Emits a {MinDelayChange} event.
+     *
+     * Requirements:
+     *
+     * - the caller must be the timelock itself. This can only be achieved by executing an operation
+     * where the timelock is the target and the data is the ABI-encoded call to this function.
```

Comment:
```suggestion
     * - the caller must be the timelock itself. This can only be achieved by scheduling and later executing
     * an operation where the timelock is the target and the data is the ABI-encoded call to this function.
```